soconda.sh

```diff
@@ -126,8 +126,10 @@ if [ "x${env_check}" = "x" ]; then
     echo "# condarc for soconda" > "${CONDA_PREFIX}/.condarc"
     echo "channels:" >> "${CONDA_PREFIX}/.condarc"
     echo "  - conda-forge" >> "${CONDA_PREFIX}/.condarc"
+    echo "  - nodefaults" >> "${CONDA_PREFIX}/.condarc"
     echo "changeps1: true" >> "${CONDA_PREFIX}/.condarc"
     echo "env_prompt: '({name}) '" >> "${CONDA_PREFIX}/.condarc"
+    echo "solver: libmamba" >> "${CONDA_PREFIX}/.condarc"
```

Comment:
Again, I believe this option is not used, and that the setting must be done in the condarc of the base environment...

Reply:
All `.condarc` in [search path](https://docs.conda.io/projects/conda/en/latest/user-guide/configuration/use-condarc.html#searching-for-condarc) will be merged. If there is any conflict, for example if user does not install conda from the helper script, and in their `~/.condarc` the solver is `solver: classic`, then `$CONDA_PREFIX/.condarc` will [overwrite](https://docs.conda.io/projects/conda/en/latest/user-guide/configuration/use-condarc.html#conflict-merging-strategy) the old option. So if the conda is installed with the helper script, this doesn't make any difference.

Reply:
Ok, thanks for clarifying.